addon_toolkit/interfaces/citation.py

```diff
@@ -38,17 +38,17 @@ class ItemType(Enum):
 
 @dataclasses.dataclass(slots=True)
 class ItemResult:
-    item_id: int
+    item_id: str
     item_name: str
-    item_type: ItemType
+    item_type: str
```

Comment:
why is `item_type` not of type `ItemType` ?

Reply:
the **Mendeley** has a lot of different document types such as **book, journal** etc and our `ItemType.DOCUMENT` was not suitable for it to work properly that's why i made is as `str`.
When we with folks will be pushing Zotero's implementation we will discuss this also, i think we might change the `ItemType.DOCUMENT` logic a little